Gemfile.lock

```diff
@@ -2,75 +2,80 @@ PATH
   remote: .
   specs:
     slanger (0.6.0)
-      activesupport (~> 4.2.1)
```

Comment:
Just actually looked at the builds - we can probably drop support for older ruby versions. 
I'd say it makes sense to merge this if you remove the failing older rubies from CI, add the latest ones, and update the docs to say what ruby version this supports.